lib/webauthn/attestation_statement/base.rb

```diff
@@ -147,6 +151,32 @@ def raw_subject_key_identifier
         OpenSSL::ASN1.decode(ext_value.value).value
       end
 
+      def valid_signature?(authenticator_data, client_data_hash, public_key = nil)
+        public_key ||= attestation_certificate.public_key
```

Comment:
Any reason not to just set this as the default instead of `nil` in the method signature?

Reply:
No, a syntactic style of my own, but I'll update if you like using it from the method signature directly 👍